dags/legacy_test/tests/pytorch/r2.5/common.libsonnet

```diff
@@ -108,9 +108,13 @@ local volumes = import 'templates/volumes.libsonnet';
         pip install torch_xla[tpu] -f https://storage.googleapis.com/libtpu-releases/index.html
         pip install pillow
         if [ ! -d "pytorch" ]; then
-          git clone --depth=1 https://github.com/pytorch/pytorch.git
+          git clone --depth=1 -b v2.5.0-rc9 https://github.com/pytorch/pytorch.git
+          cd pytorch
+        else
+          cd pytorch
+          git fetch origin release/2.5
+          git checkout -b release/2.5 origin/release/2.5
```

Comment:
Curious why we need the pytorch git repo here?  We have the `pip install torch==2.5.0 --index-url https://download.pytorch.org/whl/test/cpu` above. That seems need to be the `-rc9` wheel build instead.

Reply:
yes, I have the same curious too, and my answer is we might need PyTorch repo/code in CI test

Reply:
found some PyTorch op tests: https://github.com/pytorch/xla/blob/f71c02d1f457d58371e013632efb016c01bd1866/test/run_tests.sh#L143C1-L158C2

Reply:
Got it, thanks!